app/controllers/responsiblepeople/PersonNonUKPassportController.scala

```diff
@@ -55,10 +56,12 @@ class PersonNonUKPassportController @Inject()(
                          edit: Boolean, flow: Option[String] )(implicit authContext:AuthContext, request: Request[AnyContent]) = {
     (for {
       cache <- result
+      phase2Changes <- Some(appConfig.phase2ChangesToggle)
       rp <- getData[ResponsiblePerson](cache, index)
-    } yield rp.dateOfBirth.isDefined && edit match {
-      case true => Redirect(routes.DetailedAnswersController.get(index, flow))
-      case false => Redirect(routes.DateOfBirthController.get(index, edit, flow))
+    } yield (rp.dateOfBirth.isDefined && edit, phase2Changes) match {
```

Comment:
Why not use `if phase2Changes` similar to elsewhere?

Reply:
Using a tupl resulted in simpler to read code in this instance IMHO

Reply:
Commented below- still not convinced
 |
\/